packages/authorization-process/src/utilities/errorMappers.ts

```diff
@@ -104,3 +104,9 @@ export const createKeysErrorMapper = (error: ApiError<ErrorCodes>): number =>
     .with("notAllowedPrivateKeyException", () => HTTP_STATUS_FORBIDDEN)
     .with("keyAlreadyExists", () => HTTP_STATUS_FORBIDDEN)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const getClientKeyErrorMapper = (error: ApiError<ErrorCodes>): number =>
+  match(error.code)
```

Comment:
we need to handle also `organizationNotAllowedOnClient`

Reply:
Updated